Makefile

```diff
@@ -68,6 +71,7 @@ vendorinstall: ## Installs all protobuf dependencies with go-vendorinstall
 	go-vendorinstall github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
 	go-vendorinstall golang.org/x/tools/cmd/goimports
 	go get -u github.com/jteeuwen/go-bindata/...
+	go get -u github.com/swaggo/swag/cmd/swag
```

Comment:
I would lock the version in the toml, so we do not have undesired behaviors with upgrades.